test/binCases/config-location/find-recursively/packages/foo/find-recursively.test.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+const { run } = require("../../../../../testUtils");
+
+test("find-recursively", () => {
+	const { code, stdout, stderr } = run(__dirname, [
+		"--output-filename",
+		"[name].js",
+		"--output-chunk-filename",
+		"[id].chunk.js",
+		"--target",
+		"async-node",
+		"--mode",
+		"production"
+	]);
+	expect(code).toBe(0);
+	expect(stdout).toEqual(expect.anything());
```

Comment:
You should use the `extractHash` utility.

Reply:
@evenstensberg @ematipico So should I use `extractHash` here? I think I fixed the broken test cases except this one.